docs/changelog.md

```diff
@@ -1,3 +1,8 @@
+# v0.5.0 (2018-08-14)
+
+* Add support for setting approvers with the `-r,--approvers` option in `tsrc push` (GitLab Enterprise Edition only).
```

Comment:
Is `-r` a good idea?

Reply:
yes. Let's use it first and bikeshed later